### PR TITLE
Document and improve usability of target-number feature

### PR DESCRIPTION
**Pull request by prescod**
The target-number command line was not documented. Now it is.

While documenting it, I thought it would be better if it read in the same order as English: "50 Account"(s). Both orders are supported in the code, however.

### CODE REVIEW

**Review by Br4nd0R — DISMISSED** [review]
Looks good!

File: docs/index.md [hunk 1]
```diff
@@ -884,7 +884,31 @@ Options:
   
 ```
 
-## CSV Output
+### Scaling up recipe execution
+
+From the command line you can control how many rows a recipe generates by telling
+Snowfakery how many times to execute it. You do this by specifying a "target count"
```
Comment (davisagli) [inline docs/index.md:890]: Based on the explanation below, "how many times to execute it" is a bit misleading.

File: snowfakery/cli.py [hunk 2]
```diff
@@ -267,6 +283,18 @@ def validate_options(
         )
 
 
+def stopping_criteria_from_target_number(target_number):
+    "Deconstruct a tuple of 'str number' or 'number str' and make a StoppingCriteria"
+
+    # 'number str' is the official format so the other one can be deprecated one day.
```
Comment (davisagli) [inline snowfakery/cli.py:289]: Isn't putting them in a canonical order already accomplished by int_string_tuple?

File: docs/index.md (hunk 1 above)
Comment (prescod) [reply docs/index.md]: 99d3fc3

File: snowfakery/cli.py (hunk 2 above)
Comment (prescod) [reply snowfakery/cli.py:289]: I'll merge the embedding API next.

https://github.com/SFDO-Tooling/Snowfakery/compare/feature/embedding-api?expand=1

**Review by davisagli — APPROVED** [review]

**Pull request merged**